.gitignore

```diff
@@ -112,3 +112,6 @@ prime/
 
 # Manpage
 /man
+
+# Tags file, simply because it's too large to commit (200MB with node modules)
```

Comment:
Where does this file come from? I have never seen it in my instance.

Reply:
I've created it manually with `ctags`.

Reply:
You can store this in `.git/info/exclude`.